[2024-12-05T14:43:59.439+0100] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-05T14:43:59.444+0100] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_warehouse_etl_standard.load_fact_table manual__2024-12-05T13:43:48.500429+00:00 [queued]>
[2024-12-05T14:43:59.446+0100] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_warehouse_etl_standard.load_fact_table manual__2024-12-05T13:43:48.500429+00:00 [queued]>
[2024-12-05T14:43:59.446+0100] {taskinstance.py:2866} INFO - Starting attempt 1 of 3
[2024-12-05T14:43:59.454+0100] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_fact_table> on 2024-12-05 13:43:48.500429+00:00
[2024-12-05T14:43:59.460+0100] {standard_task_runner.py:72} INFO - Started process 61970 to run task
[2024-12-05T14:43:59.463+0100] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_warehouse_etl_standard', 'load_fact_table', 'manual__2024-12-05T13:43:48.500429+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/data_warehouse_etl_dag.py', '--cfg-path', '/var/folders/4k/bhptqqm131v6f034phck6fc80000gn/T/tmpift1z0sf']
[2024-12-05T14:43:59.465+0100] {standard_task_runner.py:105} INFO - Job 85: Subtask load_fact_table
[2024-12-05T14:43:59.490+0100] {task_command.py:467} INFO - Running <TaskInstance: data_warehouse_etl_standard.load_fact_table manual__2024-12-05T13:43:48.500429+00:00 [running]> on host sijibomis-mbp
[2024-12-05T14:43:59.509+0100] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineering' AIRFLOW_CTX_DAG_ID='data_warehouse_etl_standard' AIRFLOW_CTX_TASK_ID='load_fact_table' AIRFLOW_CTX_EXECUTION_DATE='2024-12-05T13:43:48.500429+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-05T13:43:48.500429+00:00'
[2024-12-05T14:43:59.509+0100] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-05T14:43:59.581+0100] {load.py:124} INFO - Loaded 456 fact table records
[2024-12-05T14:43:59.581+0100] {python.py:240} INFO - Done. Returned value was: True
[2024-12-05T14:43:59.585+0100] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-05T14:43:59.585+0100] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_warehouse_etl_standard, task_id=load_fact_table, run_id=manual__2024-12-05T13:43:48.500429+00:00, execution_date=20241205T134348, start_date=20241205T134359, end_date=20241205T134359
[2024-12-05T14:43:59.613+0100] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-05T14:43:59.618+0100] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-05T14:43:59.618+0100] {local_task_job_runner.py:245} INFO - ::endgroup::
